# The distance from LRF center to furthest vertice of robot is 0.23 so we
# make this distance bigger by 0.05 so we also take care of the movement 
# speed
high_security_distance : 0.28
# This the distance to the opposite side of the wall we are following.
# We make it smaller so that we can fit into tighter corners
low_security_distance : 0.14
# This distance is 0.4 so that there is 0.12 space for the robot to march
# forward
wall_follow_distance : 0.4
# 0.4 assures that we can take tight turns and also that we never hit the
# wall given the specified security distance
linear_velocity : 0.4
# 1 is enough for the turn to have the desired effect without getting into
#complicated situations
angular_velocity : 1
# Defines the starting angle of the right robot range
right_limit: 75
# Defines the starting angle of the left robot range
left_limit: 165
# Topic to publish messages to robot
publish_topic: "cmd_vel"
# Topic to get data from LRF
laser_topic: "base_scan"
# Topic to get data from circle detector
circle_topic: "circle_detect"
# Variable to detect that we are in simulation to change hit circle mode
simulation: true
